pkg/util/util.go

```diff
@@ -1118,3 +1119,27 @@ func sliceContainsString(str string, slice []string) bool {
 	}
 	return false
 }
+
+func AddFileToIgnoreFile(gitIgnoreFile, filename string) error {
+	return addFileToIgnoreFile(gitIgnoreFile, filename, filesystem.DefaultFs{})
+}
+
+func addFileToIgnoreFile(gitIgnoreFile, filename string, fs filesystem.Filesystem) error {
+	var data []byte
+	file, err := fs.OpenFile(gitIgnoreFile, os.O_APPEND|os.O_RDWR, 0600)
```

Comment:
Better to use a const for file access mode 0600 instead of hardcoding

Reply:
done